src/ui/profile_preferences.ui

```diff
@@ -55,6 +55,19 @@
        <string>General</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_19">
```

Comment:
Ah man, we can't have scrolling tabs - that would be a sign of a serious problem that we've got too much crap.

We could perhaps make Colour view just accessible from the Main display tab - same with Mapper colors to just be accessible from the Mapper tab. However a redesign of the profile preferences is not in the plan for 3.3 and there's no time to do it either, at least not from me. 

Normal users won't be changing what IRC server on what port should Mudlet connects, so it sounds OKish to me to keep it in Special Options, especially since space-wise it fits in very well.

The issue is that I've already had to increase the size of the profile preferences in order to fit the Editor tab in and making the dialog even wider is making it really rather unwieldy.